### PR TITLE
New Presence: Novel Mania

### PR DESCRIPTION
**Pull request by Shulchl**
Novel Mania is a website entirely focused on everything kind of novels, it has just launched a brand new site from scratch and I want to kindly ask for a premid extensions to implement on it. Though I'm not a professional programmer, I've done my best by having other sites extensions as a reference.

### At home theres 2 kinds of states: 
image
image

### At the Index: 
image

### Reader (got the % from beneath):
image

### User profile:
image

### Basic pages:
image

### CODE REVIEW

**Comment by Shulchl** [comment]
Correction: _Reader (got the % from **beneath**):_

File: websites/N/Novel Mania/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "Shuichi",
+    "id": "179440483796385792"
+  },
+  "service": "Novel Mania",
+  "description": {
+    "en": "Read webnovels in Portuguese anywhere and anytime.",
+    "pt_BR": "Leia webnovels em português a qualquer momento e a qualquer hora."
+  },
+  "url": "novelmania.com.br",
+  "version": "1.0.1",
```
Comment (doomlerd) [inline websites/N/Novel Mania/dist/metadata.json:13]: 1.0.0

Comment (Shulchl) [reply websites/N/Novel Mania/dist/metadata.json]: Done.

**Comment by doomlerd** [comment]
Fix the issues reported by DeepScan.

**Review by doomlerd — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**